rllib/evaluation/rollout_worker.py

```diff
@@ -406,7 +406,7 @@ def make_env(vector_index):
         if (ray.is_initialized() and
                 ray.worker._mode() != ray.worker.LOCAL_MODE):
             # Check available number of GPUs
-            if not ray.get_gpu_ids():
+            if not ray.get_gpu_ids(as_str=True):
```

Comment:
Don't we need a corresponding code change to convert these to ints somewhere or do they just get passed into something that understands them as strings? Same with `rllib/policy/torch_policy.py`

Reply:
oh for RLlib, I think this is just an existence check (specific values don't seem to actually be used).